superbuild/CMakeLists.txt

```diff
@@ -55,7 +53,16 @@ list(APPEND external_projects
   DCMTK 
   QtDCM 
   dtk
-  )
+  jsoncons
+  asio
+  websocketpp
+  eigen
+  gvirtualXRay
+  qwt 
+  mmg 
+  tetgen
+  #openssl
```

Comment:
to remove

Reply:
openssl have to uncommented for some plugins but since its compilation needs **make -j1**, it is commented for the other project.